pom.xml

```diff
@@ -37,12 +37,20 @@
             <id>sk89q-repo</id>
             <url>https://maven.enginehub.org/repo/</url>
         </repository>
+        <repository>
+            <id>mvnrepository</id>
+            <url>https://mvnrepository.com/</url>
+        </repository>
+        <repository>
+            <id>papermc</id>
+            <url>https://repo.papermc.io/repository/maven-public/</url>
+        </repository>
     </repositories>
     <dependencies>
         <dependency>
-            <groupId>org.spigotmc</groupId>
-            <artifactId>spigot-api</artifactId>
-            <version>1.16.5-R0.1-SNAPSHOT</version>
+            <groupId>io.papermc.paper</groupId>
```

Comment:
Should we really change this? Because we might be only depending then on paper, not on bukkit servers.
We just won't know...

Reply:
From bstats: https://bstats.org/plugin/bukkit/BetterTeams/7855
10% of our user base use spigot (not paper) so we cant really just drop support